### PR TITLE
Polars in hvplot.plot: avoid registering the hvplot accessor and simple test

### PR DESCRIPTION
**Pull request by maximlt**
Follow-up of  to add a simple test and to avoid registering the hvplot accessor, as I don't think this was intended.

### CODE REVIEW

File: hvplot/tests/testplotting.py [hunk 1]
```diff
@@ -58,3 +68,10 @@ def setUp(self):
             raise SkipTest('entrypoints for plotting.backends was added '
                            'in pandas 0.25.1')
         pd.options.plotting.backend = 'hvplot'
+
+
+@pytest.mark.skipif(skip_polar, reason="polars not installed")
+def test_plot_supports_polars():
+    dfp = pl.DataFrame(pd._testing.makeDataFrame())
```
Comment (Hoxbro) [inline hvplot/tests/testplotting.py:75]:
```suggestion
def test_plot_supports_polars():
    pl = pytest.importorskip("polars")
    dfp = pl.DataFrame(pd._testing.makeDataFrame())
```

Alternative way (also needs import removal also)

**Review by Hoxbro — APPROVED** [review]
A tiny comment, but nothing that blocks this PR.

File: hvplot/tests/testplotting.py (hunk 1 above)
Comment (maximlt) [reply hvplot/tests/testplotting.py]: Made the change, even if I'd say that having the marker above the test definition helps distinguish tests when looking at many of them, while an inline skip is easy to miss.

Comment (Hoxbro) [reply hvplot/tests/testplotting.py]: Yep, I wouldn't have had a problem if you kept what you had before.

**Pull request merged**